packit/distgit.py

```diff
@@ -291,8 +298,9 @@ def upload_to_lookaside_cache(self, archive_path: str) -> None:
     def is_archive_in_lookaside_cache(self, archive_path: str) -> bool:
         archive_name = os.path.basename(archive_path)
         try:
+            stg = "stg." if self.stage else ""
             res = requests.head(
-                "https://src.fedoraproject.org/lookaside/pkgs/"
+                f"https://src.{stg}fedoraproject.org/lookaside/pkgs/"
```

Comment:
Would it be possible to make this URL also configurable and get rid of the `stage` flag entierly?

Reply:
I have changed this line to use `dist_git_base_url` from package config, but the `stage` parameter should be used to forward info whether to use `fedpkg` or `fedpkg-stage` https://github.com/packit/packit/blob/624fb4c699b316e8ae5606e6ef7f50aa70604af7/packit/fedpkg.py#L45

Reply:
I see, thanks!

Probably would be nice to have this also configurable instead of hard-coded, but this is outside the scope of this PR, I think.